### ISSUE TITLE
Dashboard Components

### ISSUE THREAD

**Issue by cgarwood** [cgarwood/hacontrolpoint#11]
The HEIMA design has the dashboard split into 3 main containers. Top left, (Current Inside Temperature, swipeable). Top right (Next Scheduled Task, swipeable), and Bottom (presence detection).

What all components should we have available on our dashboard? I'm thinking we should have it able to split into 4 containers (top left, top right, bottom left, bottom right), with the ability to combine both top or both bottom containers into one larger container.

Dashboard Component Ideas:

- Current Inside Temperature
- Current Outside Temperature
- Current Weather Conditions
- Weather Forecast
- Presence Detection
- System Status (HASS systemmonitor)
- Travel Times (HASS travel components)
- Sun status (HASS sun component)
- Home Status? (some people set up switches for Vacation, Home, Away, Asleep style statuses to control automations)
- CCTV Camera Feed
- Currently Playing (HASS media_player components)
- Upcoming Appointments (Google Calendar, etc)
- Switch Panel (let people add buttons to trigger commonly used switches/scripts)

Any other ideas that should be available on the dashboard?

**Comment by jcreynolds** [comment]
If I understand correctly you are going to make the page layouts configurable based upon a quadrant style of layout?

Weather at glance would be lo/hi for Current Day and Next Day, clicking the weather widget would pull up a 10 Day Forecast?  Home Status' also could include Good Morning Good Night (Turn all lights on or off after a period of time, 30 seconds for example).

**Comment by cgarwood** [comment]
Yeah, I'm thinking quadrants would be best (but with the ability to combine either the top two or bottom two quadrants into a single larger quadrant).
